index.js

```diff
@@ -1,7 +1,10 @@
 const fs = require('fs');
+const Hapi = require('hapi');
```

Comment:
Curious why hapi and not express/plain http/etc. ?

Reply:
I've got inspiration from serverless offline, I'm not super familiar with the node ecosystem :)